src/components/LHNOptionsList/OptionRowLHN.js

```diff
@@ -124,6 +132,13 @@ function OptionRowLHN(props) {
         );
     };
 
+    const emojiCode = lodashGet(optionItem, 'status.emojiCode', '');
+    const statusText = lodashGet(optionItem, 'status.text', '');
+    const statusClearAfterDate = lodashGet(optionItem, 'status.clearAfter', '');
+    const formattedDate = DateUtils.getStatusUntilDate(statusClearAfterDate);
+    const statusContent = formattedDate ? `${statusText} (${formattedDate})` : statusText;
+    const isStatusVisible = Permissions.canUseCustomStatus(props.betas) && emojiCode && ReportUtils.isShowEmojiStatus(optionItem);
```

Comment:
this line will throw an error because there is no `isShowEmojiStatus` in the `ReportUtils` module

Reply:
👍🏻